test/test_meta.py

```diff
@@ -900,11 +900,13 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
 meta_dispatch_early_skips = set({
     torch.Tensor.float_power_,
     # Errors out in one of the tests, while ProxyTensor passes...
+    torch.Tensor.cumprod_,
```

Comment:
Due to `_reduction` (used by `sum`/`prod`). `out` and `dtype` disagree. Might leave this for later.